iroh-net/src/discovery/local_swarm_discovery.rs

```diff
@@ -267,23 +267,23 @@ impl From<&Peer> for DiscoveryItem {
 
 impl Discovery for LocalSwarmDiscovery {
     fn resolve(&self, _ep: Endpoint, node_id: NodeId) -> Option<BoxStream<Result<DiscoveryItem>>> {
-        let (send, recv) = flume::bounded(20);
+        let (send, recv) = async_channel::bounded(20);
         let discovery_sender = self.sender.clone();
         tokio::spawn(async move {
             discovery_sender
-                .send_async(Message::SendAddrs(node_id, send))
+                .send(Message::SendAddrs(node_id, send))
                 .await
                 .ok();
         });
```

Comment:
does... this even make sense?  our actors are so messy!  but nothing this PR introduced...

Reply:
🤷‍♂️ I am just the search and replace machine at this point. I don't understand any of this...

Reply:
Yeah I mentioned the spawn thing in code review as well: https://github.com/n0-computer/iroh/pull/2376#discussion_r1668183933

We agreed to handle this in a followup: https://discord.com/channels/949724860232392765/955909916487463032/1259908995376877609